Header header
int32 subjectBarcode
float32 range
float32 bearing